frontend/src/components/shared/TableOfContents.vue

```diff
@@ -39,10 +40,25 @@ export default {
       default: () => [],
     },
   },
+  methods: {
+    hasActiveSubsection({ subsections }) {
+      return (subsections ?? []).map(s => s.link).includes(this.$route.hash);
+    },
+  },
 };
 </script>
 
 <style lang="scss" scoped>
+@media screen and (min-width: $tablet) {
+  #table-of-contents {
+    position: sticky;
+    top: 0;
+    align-self: flex-start;
```

Comment:
Is it the Bulma class `container` in `./frontend/src/layouts/AboutLayout.vue` and `./frontend/src/components/Documentation.vue` that makes this div a child to a flexible container?

Reply:
Good question! I think it's the `columns` class that adds the `display: flex;` style. Really nice that you spotted this as it points to an opportunity to further refactor these similar layouts.